File: CJY/BJ/No_2581.java
import java.io.*;

public class No_2581 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int M = Integer.parseInt(br.readLine());
        int N = Integer.parseInt(br.readLine());
        br.close();

        int cnt = 0; // 소수의 합
        int min = 0; // 최솟값인 소수

        for (int i = M; i <= N; i++) {
            boolean b = false;
            for (int j = 2; j < i; j++) {
                if(i % j == 0) {
                    b = true;
                    break;
                }
            }
            if(b || i == 1) continue;
            if(min == 0) min = i;
            cnt += i;
        }

        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        if(min > 0) {
            bw.write(String.valueOf(cnt));
            bw.newLine();
            bw.write(String.valueOf(min));
        } else {
            bw.write("-1");
        }
        bw.flush();
        bw.close();
    }
}

File: CJY/BJ/No_2581.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout});
let M, N;
let cnt = 0; // 소수의 합
let min = 0; // 최솟값인 소수

rl.on("line", (s) => {
    if(!M) {
        M = parseInt(s);
    } else {
        N = parseInt(s);
        for (let i = M; i <= N; i++) {
            let b = false;
            for (let j = 2; j < i; j++) {
                if(i % j === 0) {
                    b = true;
                    break;
                }
            }
            if(b || i === 1) continue;
            if(min === 0) min = i;
            cnt += i;
        }
        console.log(min > 0 ? cnt + "\n" + min : -1);
        rl.close();
    }
}).on("close", () => {process.exit();});